### PR TITLE
Handle IP Addresses in SAN

### PR DESCRIPTION
**Pull request by yschimke**
Handle IP addresses gracefully rather than ByteString hex display

### CODE REVIEW

**Comment by yschimke** [comment]
After 

```
CN: 	Idea
Serial:	0076afe089
Pin:	sha256/0f54dfa5bbbfa66b586eec5b7d0742a1cec80f2fda4b977a3b4aacb82877e32b
SAN: 	/127.0.0.1, /172.22.191.67, localhost, yschimke-pc, yschimke-pc.localdomain
OU: 	Development
Key Usage: DigitalSignature, KeyEncipherment
Ext Key Usage: serverAuth
Valid: 	2020-11-28T15:42:21Z..2021-02-26T15:42:21Z (2 months)
```

File: certifikit-cli/src/main/kotlin/app/cash/certifikit/cli/certificateOutput.kt [hunk 1]
```diff
@@ -98,6 +100,12 @@ private fun Certificate.subjectAlternativeNameValue(): List<String>? {
         it.value as List<Pair<Any, Any>>
       }
       ?.map {
-        it.second.toString()
+        if (it.second is String) {
+          it.second as String
+        } else if (it.first is BasicDerAdapter<*> && (it.first as BasicDerAdapter<*>).tag == 7L) {
```
Comment (yschimke) [inline certifikit-cli/src/main/kotlin/app/cash/certifikit/cli/certificateOutput.kt:105]: This awkwardly needs to match against CertificateAdapters.kt

```
  internal val generalNameDnsName = Adapters.IA5_STRING.withTag(tag = 2L)
  internal val generalNameUniformResourceIdentifier = Adapters.IA5_STRING.withTag(tag = 6L)
  internal val generalNameIpAddress = Adapters.OCTET_STRING.withTag(tag = 7L)
  internal val generalName: DerAdapter<Pair<DerAdapter<*>, Any?>> = Adapters.choice(
      generalNameDnsName,
      generalNameUniformResourceIdentifier,
      generalNameIpAddress,
      Adapters.ANY_VALUE
  )
```

Comment (yschimke) [reply certifikit-cli/src/main/kotlin/app/cash/certifikit/cli/certificateOutput.kt:105]: Any advice on restructuring both side of this happily accepted.

Comment (oldergod) [reply certifikit-cli/src/main/kotlin/app/cash/certifikit/cli/certificateOutput.kt:105]: Extracting first and second and giving them names would be better probably

```
?.map { (first_with_meaningful_name, second_with_meaningful_name) ->
}
```

**Review by oldergod — APPROVED** [review]

**Pull request merged**